app/workers/rapid_ready_for_decision/rrd_processor.rb

```diff
@@ -10,34 +12,53 @@ def initialize(form526_submission)
     end
 
     def run
-      assess_data
+      use_vro = Flipper.enabled?(:rrd_call_vro_service)
```

Comment:
This is where Flipper determines whether to use classic RRD, or the VRO client, and it controls two branches in this method (and this method alone).